test/e2e/support/utils.js

```diff
@@ -154,22 +154,39 @@ export async function ensureSidebarOpened() {
 }
 
 /**
- * Opens the inserter, searches for the given term, then selects the first
- * result that appears.
+ * Search for block in the global inserter
  *
  * @param {string} searchTerm The text to search the inserter for.
  */
-export async function insertBlock( searchTerm ) {
+export async function searchForBlock( searchTerm ) {
 	await page.click( '.edit-post-header [aria-label="Add block"]' );
 	// Waiting here is necessary because sometimes the inserter takes more time to
 	// render than Puppeteer takes to complete the 'click' action
 	await page.waitForSelector( '.editor-inserter__menu' );
 	await page.keyboard.type( searchTerm );
+}
+
+/**
+ * Inserts first inserter block. Inserter needs to be open.
+ */
+export async function insertFirstBlockInInserter() {
```

Comment:
Should it be insertFirstBlockFromInserter or insetFirstBlockFoundInInserter instead? It is a bit confusing at the moment. Does it need to be exposed? It isn’t referenced outside.

Reply:
Updated during the rebase.